docker/Dockerfile.gpu

```diff
@@ -0,0 +1,149 @@
+# build stage
+FROM nvidia/cuda:10.1-devel-ubuntu18.04 as builder
+ENV DEBIAN_FRONTEND noninteractive
+RUN export DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get dist-upgrade -y && \
+    apt-get install software-properties-common -y && \
+    add-apt-repository ppa:longsleep/golang-backports && \
+    apt-get update && apt-get dist-upgrade -y && \
+    apt-get install -y build-essential pkg-config autoconf gnutls-dev \
+            git curl golang-go && \
+   ## curl -O http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-repo-ubuntu1804_10.0.130-1_amd64.deb && \
+   # curl -O http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-repo-ubuntu1804_10.1.243-1_amd64.deb && \
+   # dpkg --force-confdef --force-confnew -i ./cuda-repo-ubuntu1804_10.1.243-1_amd64.deb && \
+    ## apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/7fa2af80.pub && \
+   # apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub && \
```

Comment:
does it serve any purpose to keep these commented lines here?

Reply:
not really, I'm just removed them.